content/en-us/education/battle-royale-series/coding-the-game-loop.md

```diff
@@ -182,23 +182,23 @@ With a **repeat loop**, the code in the loop will run at least once. Unlike a wh
 
 At this point, if you're not spawning as intended, try one of the following below.
 
-- `wait()` should be inside the repeat loop. Without the wait, the script will run too many times in a second, overloading Roblox Studio and causing an error.
+- `task.wait()` should be inside the repeat loop. Without the wait, the script will run too many times in a second, overloading Roblox Studio and causing an error.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
- `Library.task.wait()` should be inside the repeat loop. Without the wait, the script will run too many times in a second, overloading Roblox Studio and causing an error.
```